src/finn/transformation/fpgadataflow/insert_hook.py

```diff
@@ -82,7 +82,7 @@ def apply(self, model):
                     if n0_hook in list_supported_hooks:
```

Comment:
Currently there is only 1 hook in the `list_supported_hooks`, so this if-branch is always taken (and the if-statement on the next line is exactly the same).
But again, for the future if we would plan to extend this, it makes sense.